vignettes/datatable-keys-fast-subset.Rmd

```diff
@@ -108,7 +108,7 @@ Instead, in *data.tables* we set and use `keys`. Think of a `key` as **superchar
 
 #### Keys and their properties {#key-properties}
 
-1. We can set keys on *multiple columns* and the column can be of *different types* -- *integer*, *numeric*, *character*, *factor*, *integer64* etc. *list* and *complex* types are not supported yet.
+1. [ ] We can set keys on *multiple columns* and the column can be of *different types* -- *integer*, *numeric*, *character*, *factor*, *integer64* etc. *list* and *complex* types are not supported yet.
```

Comment:
why `[ ]`?

Reply:
not sure how it got there, addressed now by [7f4edd4](https://github.com/Rdatatable/data.table/pull/5402/commits/7f4edd4f94a1db43bfc04b6e770b5a1ee88e35bc)